pom.xml

```diff
@@ -351,7 +351,7 @@
             <dependency>
                 <groupId>com.facebook.presto.hadoop</groupId>
                 <artifactId>hadoop-apache2</artifactId>
-                <version>2.7.3-1-t</version>
```

Comment:
Check what was the version used for 167t-0.7 and 0.6 to make sure we use open source version and not our version

Reply:
It was changed from 0.10(167t-0.6) to 2.7.3-1-t(in 167t-0.7).  For this release we are using open source version.